src/content/docs/de/guides/environment-variables.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Umgebungsvariablen verwenden
+description: Lerne, wie du Umgebungsvariablen in einem Astro-Projekt verwenden kannst.
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+Astro nutzt die in Vite eingebaute Unterstützung für Umgebungsvariablen und lässt dich [mit jeder seiner Methoden](https://vitejs.dev/guide/env-and-mode.html) mit ihnen arbeiten.
+
+Beachte, dass zwar _alle_ Umgebungsvariablen im serverseitigen Code verfügbar sind, aber aus Sicherheitsgründen nur Umgebungsvariablen mit dem Präfix `PUBLIC_` im clientseitigen Code verfügbar sind.
+
+```ini title=".env"
+SECRET_PASSWORD=password123
+PUBLIC_ANYBODY=there
+```
+
+In diesem Beispiel ist `PUBLIC_ANYBODY` (zugänglich über `import.meta.env.PUBLIC_ANYBODY`) sowohl im Server- als auch im Client-Code verfügbar, während `SECRET_PASSWORD` (zugänglich über `import.meta.env.SECRET_PASSWORD`) nur serverseitig verfügbar ist.
+
+:::caution
+`.env`-Dateien werden nicht in [Konfigurationsdateien](/de/guides/configuring-astro/) geladen.
+:::
+
+## Standard-Umgebungsvariablen
+
+Astro enthält ein paar Umgebungsvariablen, die sofort einsatzbereit sind:
+
+- `import.meta.env.MODE`: Der Modus, in dem deine Website läuft. Das ist `development`, wenn du `astro dev` benutzt und `production`, wenn du `astro build` benutzt.
+- `import.meta.env.PROD`: `wahr`, wenn deine Website im Produktionsmodus läuft; andernfalls `falsch`.
+- `import.meta.env.DEV`: `true`, wenn deine Website im Entwicklungsmodus läuft; sonst `false`. Immer das Gegenteil von `import.meta.env.PROD`.
+- `import.meta.env.BASE_URL`: Die Basis-URL, von der aus deine Seite geladen wird. Sie wird durch die [Konfigurationsoption `base`](/de/reference/configuration-reference/#base) bestimmt.
+- `import.meta.env.SITE`: Diese wird auf [die Option `site`](/de/reference/configuration-reference/#site) gesetzt, die in der `astro.config` deines Projekts angegeben ist.
+
+## Umgebungsvariablen setzen
+
+### `.env`-Dateien
+Umgebungsvariablen können aus `.env`-Dateien in deinem Projektverzeichnis geladen werden.
+
+Du kannst auch einen Modus (entweder `production` oder `development`) an den Dateinamen anhängen, z.B. `.env.production` oder `.env.development`, so dass die Umgebungsvariablen nur in diesem Modus wirksam werden.
+
+Erstelle einfach eine `.env` Datei im Projektverzeichnis und füge ihr einige Variablen hinzu.
+
+```ini title=".env"
+# Diese Funktion ist nur verfügbar, wenn sie auf dem Server ausgeführt wird!
+DB_PASSWORD="foobar"
+# Das wird überall verfügbar sein!
+PUBLIC_POKEAPI="https://pokeapi.co/api/v2"
+```
+
+Mehr über `.env`-Dateien findest du in der Vite-Dokumentation (https://vitejs.dev/guide/env-and-mode.html#env-files).
```

Comment:
Pointing out that this link here is broken! Could you change it so the link wraps the equivalent text in German? It's below this heading in English if that helps! 🙌 
https://docs.astro.build/en/guides/environment-variables/#env-files